src/app/components/MediaLoader/Placeholder/index.tsx

```diff
@@ -30,31 +36,56 @@ const MediaPlayerPlaceholder = ({
     guidanceMessage,
   } = mediaInfo ?? {};
 
+  const playButton = (
+    <PlayButton
+      css={styles.playButton}
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onClick={() => {}}
+      title={title}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+      guidanceMessage={guidanceMessage}
+      className="focusIndicatorRemove"
+    />
+  );
+
+  const playButtonWithSustainabilityMessage = (
+    <WithSustainabilityMessageMediaIndicator
+      title={title}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+      guidanceMessage={guidanceMessage}
+    />
+  );
+
+  const guideComponent = (
+    <Guidance
+      css={styles.guidance}
+      guidanceMessage={guidanceMessage}
+      noJsMessage={noJsMessage}
+    />
+  );
+
+  const withSustainabilityMessage = <SustainabilityMessage title={title} />;
+
   return (
     // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
     <div
       onClick={onClick}
       css={styles.placeholder}
       data-e2e="media-loader__placeholder"
+      className="placeholder"
```

Comment:
Could we rename this?
```suggestion
      className="mediaLoaderPlaceholder"
```